students/10th/kimjungho/style/login.css

```
@@ -0,0 +1,73 @@
+
+input {
+    margin: 0 auto;
+    border: 1px solid lightgray;
+    border-top-left-radius: 10px; /* 왼쪽 위 */
+    border-top-right-radius: 10px; /* 오른쪽 위 */
+    border-bottom-right-radius: 10px; /* 오른쪽 아래 */
+    border-bottom-left-radius: 10px; /* 왼쪽 아래 */

```

```suggestion
    border-radius: 10px;
```

input 태그의 네 귀퉁이에 모두 radius 10px을 주고 싶으셨군요. 이 경우 CSS의 단축속성을 이용하시면 간단하게 스타일링이 가능합니다. (참고 : https://developer.mozilla.org/ko/docs/Web/CSS/Shorthand_properties)